src/kernel/src/main.rs

```diff
@@ -215,18 +214,31 @@ fn start() -> Result<(), KernelError> {
         return Err(KernelError::MountFailed(path.into(), e));
     }
 
-    let path = VPathBuf::try_from(format!(
-        "/mnt/sandbox/{}_000/app0",
-        args.game.file_name().unwrap().to_str().unwrap()
-    ))
-    .unwrap();
+    // TODO: Check permission of these paths on the PS4.
+    let paths = [vpath!("/mnt/sandbox"), vpath!("/mnt/sandbox/pfsmnt")];
```

Comment:
I originally mkdir'd these paths too, but I though that they always exist inside tmpfs

Reply:
All files in the tmpfs will live in the memory.